### PR TITLE
Feature/v1.6.0 format and clean code

### PR DESCRIPTION
**Pull request by baizetianxia**
code review: 
format and clean code for code style

### CODE REVIEW

File: contracts/eth/strategies/ETHBaseStrategy.sol [hunk 1]
```diff
@@ -207,12 +209,12 @@ abstract contract ETHBaseStrategy is Initializable, AccessControlMixin {
     function queryTokenValueInETH(address _token, uint256 _amount)
         internal
         view
-        returns (uint256 valueInETH)
+        returns (uint256 _valueInETH)
     {
         if (_token == ETHToken.NATIVE_TOKEN) {
```
Comment (ZhangBrian) [inline contracts/eth/strategies/ETHBaseStrategy.sol:214]: use boc-contract-core/library/NativeToken replace ETHToken

Comment (ZhangBrian) [reply contracts/eth/strategies/ETHBaseStrategy.sol:214]: delete ETHToken.sol

File: contracts/eth/strategies/aura/AuraREthWEthStrategy.sol [hunk 2]
```diff
@@ -235,62 +232,60 @@ contract AuraREthWEthStrategy is ETHBaseClaimableStrategy {
...
-            bool claimIsWorth,
+            bool _claimIsWorth,
             address[] memory _rewardsTokens,
             uint256[] memory _claimAmounts
         )
     {
-        address rewardPool = getRewardPool();
-        uint256 earn = IRewardPool(rewardPool).earned(address(this));
+        address _rewardPool = getRewardPool();
+        uint256 earn = IRewardPool(_rewardPool).earned(address(this));
```
Comment (ZhangBrian) [inline contracts/eth/strategies/aura/AuraREthWEthStrategy.sol:285]: _earn

File: contracts/eth/strategies/convex/ConvexSETHStrategy.sol [hunk 3]
```diff
@@ -130,40 +130,40 @@ contract ConvexSETHStrategy is ConvexBaseStrategy {
...
-        depositArray[0] = _amounts[0];
-        depositArray[1] = _amounts[1];
+        uint256[2] memory _depositArray;
+        _depositArray[0] = _amounts[0];
+        _depositArray[1] = _amounts[1];
         // only need to safeApprove sETH
-        ICurveLiquidityPoolPayable curvePool = getCurvePool();
-        address curvePoolAddress = address(curvePool);
+        ICurveLiquidityPoolPayable _curvePool = getCurvePool();
+        address curvePoolAddress = address(_curvePool);
```
Comment (ZhangBrian) [inline contracts/eth/strategies/convex/ConvexSETHStrategy.sol:150]: _curvePoolAddress

File: contracts/eth/strategies/uniswap-v2/ETHUniswapV2Strategy.sol [hunk 4]
```diff
@@ -52,37 +51,36 @@ contract ETHUniswapV2Strategy is ETHBaseStrategy, UniswapV2LiquidityActionsMixin
...
     function lpValueInEth() internal view returns (uint256 lpValue) {
-        uint256 totalSupply = uniswapV2Pair.totalSupply();
-        (uint112 reserve0, uint112 reserve1, ) = uniswapV2Pair.getReserves();
-        console.log('reserve0:%d,reserve1:%d',reserve0,reserve1);
+        uint256 _totalSupply = uniswapV2Pair.totalSupply();
+        (uint112 _reserve0, uint112 _reserve1, ) = uniswapV2Pair.getReserves();
         uint256 lpDecimalUnit = 1e18;
-        uint256 part0 = (uint256(reserve0) * (lpDecimalUnit)) / totalSupply;
-        uint256 part1 = (uint256(reserve1) * (lpDecimalUnit)) / totalSupply;
+        uint256 part0 = (uint256(_reserve0) * (lpDecimalUnit)) / _totalSupply;
```
Comment (ZhangBrian) [inline contracts/eth/strategies/uniswap-v2/ETHUniswapV2Strategy.sol:71]: _part0、_part1

File: contracts/exchanges/adapters/ParaSwapV5Adapter.sol [hunk 5]
```diff
@@ -408,11 +383,11 @@ contract ParaSwapV5Adapter is ParaSwapV5ActionsMixin, IExchangeAdapter {
...
     returns (
-        address tokenIn_,
-        uint256 amountIn_,
-        uint256 amountOutMin_,
-        address weth_,
-        uint256[] memory pools_
+        address _tokenIn,
+        uint256 _amountIn,
+        uint256 _amountOutMin,
+        address _weth_,
```
Comment (mingshengke) [inline contracts/exchanges/adapters/ParaSwapV5Adapter.sol:389]: must be name _weth and _pools

File: contracts/usd/strategies/convex/Convex3CrvStrategy.sol [hunk 6]
```diff
@@ -88,34 +88,33 @@ contract Convex3CrvStrategy is ConvexBaseStrategy {
...
+            uint256 _depositedTokenAmount = (pool.balances(i) * _lpAmount) / _totalSupply;
+            _amounts[i] = balanceOfToken(_tokens[i]) + _depositedTokenAmount;
         }
     }
 
     function get3rdPoolAssets() external view override returns (uint256) {
         address[] memory _assets = wants;
-        uint256 thirdPoolAssets;
+        uint256 _thirdPoolAssets;
         ICurveLiquidityPool pool = ICurveLiquidityPool(curvePool);
```
Comment (mingshengke) [inline contracts/usd/strategies/convex/Convex3CrvStrategy.sol:112]: _pool

File: contracts/usd/strategies/convex/ConvexCompoundStrategy.sol [hunk 7]
```diff
@@ -90,33 +90,33 @@ contract ConvexCompoundStrategy is ConvexBaseStrategy {
         override
         returns (uint256)
     {
-        uint256[] memory depositAmounts = new uint256[](2);
-        address[] memory cTokens = new address[](2);
-        cTokens[0] = cDAI;
-        cTokens[1] = cUSDC;
+        uint256[] memory _depositAmounts = new uint256[](2);
+        address[] memory _cTokens = new address[](2);
+        _cTokens[0] = cDAI;
```
Comment (mingshengke) [inline contracts/usd/strategies/convex/ConvexCompoundStrategy.sol:95]: C_DAI

File: contracts/usd/strategies/convex/ConvexCompoundStrategy.sol [hunk 8]
```diff
@@ -90,33 +90,33 @@ contract ConvexCompoundStrategy is ConvexBaseStrategy {
...
+        _cTokens[1] = cUSDC;
```
Comment (mingshengke) [inline contracts/usd/strategies/convex/ConvexCompoundStrategy.sol:96]: C_USDC

File: contracts/usd/strategies/convex/ConvexPaxStrategy.sol [hunk 9]
```diff
@@ -102,53 +102,49 @@ contract ConvexPaxStrategy is ConvexBaseStrategy {
...
     {
-        uint256[] memory depositAmounts = new uint256[](4);
-        address[] memory yTokens = new address[](4);
-        yTokens[0] = ycDAI;
-        yTokens[1] = ycUSDC;
-        yTokens[2] = ycUSDT;
-        yTokens[3] = PAX;
+        uint256[] memory _depositAmounts = new uint256[](4);
+        address[] memory _yTokens = new address[](4);
+        _yTokens[0] = ycDAI;
```
Comment (mingshengke) [inline contracts/usd/strategies/convex/ConvexPaxStrategy.sol:107]: YC_DAI  YC_USDT YC_USDC

File: contracts/usd/strategies/convex/ConvexUsdtStrategy.sol [hunk 10]
```diff
@@ -67,23 +67,23 @@ contract ConvexUsdtStrategy is ConvexBaseStrategy {
...
+        uint256 _amount = _amounts[2];
         IERC20Upgradeable(_assets[0]).safeApprove(cDAI, 0);
         IERC20Upgradeable(_assets[0]).safeApprove(cDAI, _amounts[0]);
         ICToken(cDAI).mint(_amounts[0]);
-        uint256 cDAIBalance = balanceOfToken(cDAI);
+        uint256 _cDAIBalance = balanceOfToken(cDAI);
         IERC20Upgradeable(_assets[1]).safeApprove(cUSDC, 0);
         IERC20Upgradeable(_assets[1]).safeApprove(cUSDC, _amounts[1]);
         ICToken(cUSDC).mint(_amounts[1]);
         uint256 cUSDCBalance = balanceOfToken(cUSDC);
```
Comment (mingshengke) [inline contracts/usd/strategies/convex/ConvexUsdtStrategy.sol:78]: _cusdcBalance

File: contracts/usd/strategies/sushi/kashi/stake/SushiKashiStakeStrategy.sol [hunk 11]
```diff
@@ -165,63 +156,45 @@ contract SushiKashiStakeStrategy is BaseClaimableStrategy {
...
-            "[%s] lp amount wanted is %d, lp amount is %d",
-            this.name(),
-            _withdrawAmount,
-            balanceOfLpToken()
-        );
         // 1. withdraw from MasterChef
-        masterChef.withdraw(poolId, _withdrawAmount);
-        console.log("[%s] withdraw successfully", this.name());
+        MASTERCHEF.withdraw(poolId, _withdrawAmount);
         (, uint128 base) = _kashiPair.totalAsset();
```
Comment (mingshengke) [inline contracts/usd/strategies/sushi/kashi/stake/SushiKashiStakeStrategy.sol:161]: _base

File: contracts/usd/strategies/uniswapv3/UniswapV3Strategy.sol [hunk 12]
```diff
@@ -302,94 +292,92 @@ contract UniswapV3Strategy is BaseStrategy, UniswapV3LiquidityActionsMixin {
...
             __purge(limitMintInfo.tokenId, type(uint128).max, 0, 0);
             limitMintInfo = MintInfo({tokenId: 0, tickLower: 0, tickUpper: 0});
         }
 
-        if (baseLiquidity <= 0 && limitLiquidity <= 0) return;
+        if (_baseLiquidity <= 0 && _limitLiquidity <= 0) return;
 
         // Mint new base and limit position
         (
             int24 tickFloor,
```
Comment (mingshengke) [inline contracts/usd/strategies/uniswapv3/UniswapV3Strategy.sol:318]: _tickFloor and  _tickCeil

**Review by mingshengke — APPROVED** [review]

**Pull request merged**